fw/apm.c

```diff
@@ -1047,8 +1047,12 @@ tfw_apm_stats(void *apmref, TfwPrcntlStats *pstats)
 int
 tfw_apm_stats_global(TfwPrcntlStats *pstats)
 {
-	BUG_ON(!tfw_apm_global_data);
-	return __tfw_apm_stats(tfw_apm_global_data, pstats);
+    if (!tfw_apm_global_data) {
+        pr_warn("tfw_apm: global data not initialized, skipping stats\n");
```

Comment:
Please use our logging facilities, e.g. `T_WARN()`. But it seems this is a normal case, so maybe we should use `T_DBG` instead